### DISCUSSION TITLE
Android TV version?

### DISCUSSION THREAD

**Discussion by SamDunham** [advplyr/audiobookshelf#1458] [General]
Are there plans for an Android TV version of the client (Google Play shows it as not compatible and I can't get the apk to sideload)?

Thanks,
Sam

**Comment by itkfilelor** [comment]
Download the app on your phone and then get an APK extractor from the playstore or Fdroid.
Choose the ABS app from inside the apk extractor.
Copy the apk to you PC. 
Profit.

YMMV, phone/tablet optimized apps don't always handle well when forced to run on a tv/Nshield. I tried using tasker on my TV and it was an 'S' show.

**Reply by danielb2** [reply]
> Download the app on your phone and then get an APK extractor from the playstore or Fdroid

no, the releases are all available in the [releases section for the android app](https://github.com/advplyr/audiobookshelf-app/tags) so that is not at all necessary

> Profit


No profit, as you say below, the navigation isn't made to work with android TV. I got it to sideload fine, but it's no good

**Comment by danielb2** [comment]
I don't know how much work would be involved here, but from what I can see, it's just that the navigation fails. Using the remote and it doesn't navigate properly. For example, it's stuck in the navigation bar, and will not let me navigate to the content.
